netket/optimizer/qgt/default.py

```diff
@@ -42,6 +42,12 @@ def _is_dense_solver(solver: Any) -> bool:
 
 
 def default_qgt_matrix(variational_state, solver=False, **kwargs):
+    """
+    if variational_state is an ExactState, this performs the exact summation over the whole Hilbert space and returns exact metric tensor
+    """
```

Comment:
Is that supposed to be the docstring of this method? It seems more like a comment on the lines below.